orchestration/dashboard.py

```
@@ -300,7 +300,7 @@ def render_dashboard(self):
         try:
             redis_result = subprocess.run(["redis-cli", "ping"], check=False, capture_output=True, text=True)
             redis_status = "🟢 ONLINE" if redis_result.returncode == 0 else "🔴 OFFLINE"
-        except (FileNotFoundError, subprocess.SubprocessError, OSError):
+        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)